bin/phpstan/constants.php

```diff
@@ -0,0 +1,9 @@
+<?php
+/**
+ * Constants for PHPStan static analysis.
+ *
+ * @package speculation-rules
+ */
+
+define( 'SPECULATION_RULES_VERSION', '1.2.1' );
```

Comment:
The value doesn't matter here. Just needs to be a strong for PHPStan.

```suggestion
define( 'SPECULATION_RULES_VERSION', '0.0.0' );
```